.github/workflows/deploy.yml

```diff
@@ -7,6 +7,7 @@ on:
       - deploy-hernan
       - deploy-pablo
       - shlevy
+      - sprint-*
```

Comment:
oops, I guess I don't need this star anymore - I'm going to remove

Reply:
or should I @shlevy ? I think I need this. Going to leave this alone till I hear from you.